src/clj/witan/send/send.clj

```diff
@@ -309,9 +323,11 @@
         transitions (if (nil? modified-transition-matrix)
                       (u/transitions-map transition-matrix)
                       (u/transitions-map modified-transition-matrix))
-        transition-matrix-filtered (if (nil? modified-transition-matrix)
-                                     (filter #(= (:calendar-year %) 2016) transition-matrix)
-                                     (filter #(= (:calendar-year %) 2016) modified-transition-matrix))
+        transition-matrix-filtered (if (nil? filter-transitions-from)
+                                     nil
+                                     (if (nil? modified-transition-matrix)
+                                       (mapcat (fn [year] (filter #(= (:calendar-year %) year) transition-matrix)) filter-transitions-from)
```

Comment:
remove if and just use (or modified-transition-matrix transition-matrix) in the expression